ios/engine/KMEI/KeymanEngine.xcodeproj/project.pbxproj

```diff
@@ -1315,16 +1315,17 @@
 			);
 			inputPaths = (
 			);
+			name = "Run Script - set bundle versions";
 			outputFileListPaths = (
 			);
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
+			shellPath = "/usr/bin/env bash";
```

Comment:
Do all shellPaths need to be updated?
Like ll. 1293 and 1306 (there may be more)